WordPress-VIP-Go/ruleset.xml

```diff
@@ -159,6 +159,10 @@
 		<severity>5</severity>
 		<message>Having more than 100 posts returned per page can lead to severe performance problems.</message>
 	</rule>
+	<rule ref="WordPressVIPMinimum.Filters.RestrictedHook.HighTimeout">
```

Comment:
These four lines do nothing - the type is already set as a `Warning`, and the severity of 5 is the default anyway.

Reply:
@GaryJones Then should I just add the ruleset `WordPressVIPMinimum.Filters.RestrictedHook` as a whole or is it not necessary at all?

Reply:
It shouldn't be necessary, as [line 6](https://github.com/Automattic/VIP-Coding-Standards/pull/281/files#diff-0668f43b46ab1d5d149e12c1529486b8R6) adds the whole of `WordPressVIPMinimum` ruleset / sniffs anyway.

IMO the only thing that should appear here, are _amendments_ - a change of message or severity or type, for VIP Go, compared to the attributes set in the Sniff which apply to WPCOM.